### ISSUE TITLE
Question about input data

### ISSUE THREAD

**Issue by lizhuoq** [198808xc/Pangu-Weather#51]
If I perform inference using the ONNX data provided in your repository, besides converting the data from NC format to the corresponding NPY format, do I also need to normalize the data to have a mean of 0 and a standard deviation of 1? I noticed in the pseudocode for the inference function that it requires input data to be in the form of mean 0 and standard deviation 1, but this is not mentioned in your readme file. I would appreciate your response. Thank you.

**Comment by 198808xc** [comment]
No, only data format conversion is required. The ONNX file contains both normalization and de-normalization, which is different from the pseudocode (in the pseudocode, the Model() functions only contain normalization but not de-normalization).
